docs/guide/advanced/filled-icons.md

```diff
@@ -0,0 +1,13 @@
+# Filled Icons
+
+Fills are officially not supported.
+However all SVG properties are available on all icons.
+Fill can still be used, and will work fine on certain icons.
+
+Example with stars:
+
+<!-- Code Example with stars -->
+
+## Will Lucide have fills in the future?
+
+This feature is requested several times, so maybe it will be considered in future projects.
```

Comment:
```suggestion
This feature has been requested several times, so maybe it will be considered in future projects.
```

If we had an open PR for the discussion, we could say `and discussion is heppening at #999` instead of `so maybe it will be considered in future projects.`

Reply:
yeah good one, we have this issue. So will link that one!

Reply:
Added it!